.code-samples.meilisearch.yaml

```diff
@@ -46,7 +46,10 @@ delete_one_document_1: |-
 delete_documents_1: |-
   $client->index('movies')->deleteDocuments([23488, 153738, 437035, 363869]);
 search_1: |-
-  $client->index('movies')->search('american ninja');
+  $searchResults = $client->index('movies')->search('american ninja')
+  $searchResults->getHits();
+  $searchResults->getFacetsDistribution();
+  $searchResults->getRaw(); // The JSON response returned by MeiliSearch
```

Comment:
> But in my humble opinion, in a very simple approach, I would keep the following line and not the raw one as per default PHP users will expect a PHP class instance.

>  $searchResults = $client->index('movies')->search('american ninja')

The problem with this: the user does not even know what are the main methods exist.

We cannot follow algolia or elastic -> they don't return an instance class like us. They provide "simple" wrapper.

I did this suggestion because I don't want our user to be lost with a huge class he/she can't even read when trying to display it.
But I really open to improvement, I felt a little bit confused about what to do here.

Reply:
Mmmh, if we had to add it to the search API references I think we should adapt every SDK to showcase the same possibilities. 

Also, if we add your example, I think context is needed. For example:

```php
   // Do a search
   $searchResults = $client->index('movies')->search('american ninja')

  // Get results in array
  $hits = $searchResults->getHits();
  $facetDistribution = $searchResults->getFacetsDistribution();

  // Get results in JSON
  $searchResults->getRaw(); // JSON encoded answer of MeiliSearch, see response bellow
```

Reply:
```php
   // Do a search
   $searchResults = $client->index('movies')->search('american ninja')

  // Get results in an Array using a getter
  $hits = $searchResults->getHits();

  // Get the decoded response of MeiliSearch, see response below
  $response = $searchResults->getRaw();
```